subprojects/distributions/src/integTest/groovy/org/gradle/SrcDistributionIntegrationSpec.groovy

```diff
@@ -50,7 +50,7 @@ class SrcDistributionIntegrationSpec extends DistributionIntegrationSpec {
             inDirectory(contentsDir)
             usingExecutable('gradlew')
             withTasks('binZip')
-            withArguments("-Djava9Home=${System.getProperty('java9Home')}", "-D${PLUGIN_PORTAL_OVERRIDE_URL_PROPERTY}=${gradlePluginRepositoryMirrorUrl()}")
```

Comment:
Are we running this distribution tests only on Java 9? I guess we need to run the `BuildDistributions` build on Java 9, right: https://github.com/gradle/gradle/blob/8da243f7fc2f48059dfcd9bffc527c714ca86ae1/.teamcity/Gradle_Check/configurations/BuildDistributions.kt#L13

Reply:
Yes, I think we should change `BuildDistributions` to Java 9 because our whole build is running on Java 9 now and it doesn't make any sense to continue testing it on 8. I'll change that on `teamcity` branch.